tests/queries/0_stateless/02870_per_column_settings.sql

```diff
@@ -0,0 +1,78 @@
+-- Tags: no-random-merge-tree-settings
+
+-- Tests column-level settings for MergeTree* tables
+
+DROP TABLE IF EXISTS tab;
+
+CREATE TABLE tab
+(
+    id UInt64,
+    long_string String SETTINGS (min_compress_block_size = 163840, max_compress_block_size = 163840),
+    v1 String,
+    v2 UInt64,
+    v3 Float32,
+    v4 Float64
+)
+ENGINE = ReplicatedMergeTree('/clickhouse/tables/{database}/tab/2870', 'r1')
+ORDER BY id
+SETTINGS min_bytes_for_wide_part = 1;
```

Comment:
Is there a reason why compact parts are not tested?

Reply:
Column settings are applied for specific column, but for compact parts all columns are in a single file, so I think it column level settings doesn't make sense for compact parts.